test/types/holder.go

```diff
@@ -21,12 +21,6 @@ import (
 // KogitoServiceHolder Helper structure holding informations which are not available in KogitoService
 type KogitoServiceHolder struct {
 	v1alpha1.KogitoService
-	Infinispan struct {
-		Username string
-		Password string
-	}
-	EnablePersistence bool
-	EnableEvents      bool
```

Comment:
Maybe we can get rid of the holder now and uses directly the KogitoService? Or do you want to keep it just in case we need to add things here?

Reply:
I would keep it for now, for the case if we need it in future.